ooniapi/services/ooniprobe/tests/test_bouncer.py

```diff
@@ -0,0 +1,14 @@
+"""
+Integration test for OONIProbe API
+"""
+
+
+def test_get_bouncer_nettests(client):
+    version = "1"
+
+    r = client.post("/bouncer/net-tests", json={"net-tests": [{"name": "foo", "version": version}]})
+    j = r.json()
+    assert "net-tests" in j
+    for v in j["net-tests"]:
+        for x in ["collector", "input-hashes", "name", "test-helpers", "test-helpers-alternate", "version"]:
+            assert x in v
```

Comment:
Can we also port the other tests that are here: https://github.com/ooni/backend/blob/5763024ff0859fa335971080b99f6ae89766160f/api/tests/integ/test_probe_services.py#L316?

Mostly to be sure we're not breaking the API

Reply:
I would rather go with the safe route in this one, let's change it to 400.
I was looking into how this can be done and it seems like parsing the request manually is the way to go in this case, sorry for asking you to change it in another comment

We could do something like 
```
    try:
        j = await req.json()
        m = NetTestsRequest(**j)
    except pydantic.ValidationError as e:
        raise HTTPException(400, detail=e.errors())
```

(or you could manually parse the json like you had before, any choice is good to me) 

It would mess the api docs for this endpoint but in this case it doesn't really matter since this endpoint is not used by new clients and we keep it just for backwards compatibility